app/models/vacancy.rb

```diff
@@ -6,7 +6,15 @@ class Vacancy < ApplicationRecord
 
   friendly_id :slug_candidates, use: %w[slugged history]
 
-  array_enum job_roles: { teacher: 0, leadership: 1, sen_specialist: 2, nqt_suitable: 3 }
+  # Each vacancy must have *exactly* one primary job role. It may have zero or multiple
+  # additional job roles. Legacy vacancies *may* have more than one primary job role as
+  # we used to allow multiple.
+  # TODO: This is a compromise to keep changes to the data model minimal for now. Once
+  # the legacy vacancies are gone, we should refactor the data model.
+  PRIMARY_JOB_ROLES =  { teacher: 0, leadership: 1, education_support: 4, sendco: 5 }.freeze
+  ADDITIONAL_JOB_ROLES = { sen_specialist: 2, nqt_suitable: 3 }.freeze
```

Comment:
These are the legacy job roles right, perhaps LEGACY_JOB_ROLES makes more sense as a name then??

Reply:
They're not legacy, they are still applicable - the difference is that every job now _must_ have exactly one of `PRIMARY_JOB_ROLES`, and it _may_ have none, one, or both of the `ADDITIONAL_JOB_ROLES` on top 💡

Reply:
`sen_specialist` gets renamed to the new thing in a later commit